mlos_bench/mlos_bench/tests/launcher_in_process_test.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests to check the launcher and the main optimization loop in-process.
```

Comment:
Was the intention for this to be able to attach to the process for debugging?
I think we might have an example of that already ...

Reply:
Nm, I'd used something like this in a few cases where the Windows VSCode pytest integration wasn't allowing be to do the usual "debug test".

```python
# rest of the test file ...

if __name__ == '__main__':
    pytest.main([__file__, "-n0"])
```

Then you can just debug the whole file as it's own executable.  Not sure if that's relevant here though.